bokehjs/src/lib/models/tools/edit/line_edit_tool.ts

```diff
@@ -0,0 +1,152 @@
+import {PanEvent, TapEvent} from "core/ui_events"
+import {GlyphRenderer} from "../../renderers/glyph_renderer"
+import {LineTool, LineToolView} from "./line_tool"
+import * as p from "core/properties"
+import {bk_tool_icon_line_edit} from "styles/icons"
+import {LineGLGlyph} from "models/glyphs/webgl"
```

Comment:
This is the source of problems with CI. You should use `Line` glyph from `models/glyphs/line`. `*GLGlyph` are an internal affair and should not be referenced outside webgl rendering engine.

Reply:
Ahh great! thanks